tutorials/tutorials.batch

```diff
@@ -13,98 +13,74 @@
 ~/bin/highlight-pointer -r 10 --key-quit q &
 mkdir /tmp/msui_tutorials
 export MSUI_CONFIG_PATH=/tmp/msui_tutorials
+export XDG_CONFIG_PATH=/tmp/msui_tutorials
```

Comment:
XDG_CONFIG_PATH is not used by MSS yet, AFAIK. You probably meant XDG_CACHE_HOME? I'd also prefer if this was a proper random temporary directory created with `mktemp -d`.